libraries/rush-lib/src/cli/actions/DeployAction.ts

```diff
@@ -4,16 +4,15 @@
 import * as path from 'path';
 import type { IPackageJson } from '@rushstack/node-core-library';
 import { CommandLineFlagParameter, CommandLineStringParameter } from '@rushstack/ts-command-line';
+import type { DeployManager, IDeployProjectConfiguration } from '@rushstack/deploy-manager/lib/DeployManager';
 
 import { BaseRushAction } from './BaseRushAction';
 import { RushCommandLineParser } from '../RushCommandLineParser';
```

Comment:
I'm guessing this should be a type-only import, since I'm pretty sure this file is imported by `RushCommandLineParser`?